docker-compose.yml

```diff
@@ -1,44 +1,36 @@
-version: '1.0'
+version: '1.2'
```

Comment:
version könnten wir updaten v1 ist deprecetad [laut docs](https://docs.docker.com/compose/compose-file/compose-versioning/)

Reply:
hab auf 3.8 hochgestuft, an sich is alles gleich nur das production tag hat sich hinter das image verschoben